ui-modules/utils/quick-launch/quick-launch.js

```diff
@@ -276,7 +281,7 @@ export function quickLaunchDirective() {
                         newApp[BROOKLYN_CONFIG] = newConfig;
                     }
                 }
-                
+
                 // prefer to use the actual yaml input, but if it's not possible
                 let tryMergeByConcatenate =
                     Object.keys(newApp).length ?
```

Comment:
Catch block might be required for the Promise in `showEditor` function. Find below.

Reply:
Added a simple `console.error` since the typical `model.deployError` prop serves a different purpose and this case is controlled enough that it should generally not be able to fail (Form > yaml text generation). Further ideas would be welcome though.

Reply:
Agree, good to have log at least.